tidy3d/components/geometry.py

```diff
@@ -1009,6 +1132,10 @@ def from_gds(  # pylint:disable=too-many-arguments
             Length scale used in GDS file in units of MICROMETER.
             For example, if gds file uses nanometers, set ``gds_scale=1e-3``.
             Must be positive.
+        dilation : float = 0.0
+            Dilation of the cross-section by a distance
```

Comment:
just use the same as the field descriptions here (also periods at end).

Reply:
fixed